.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: Scala CI
+
+on:
+  push:
+    branches: [ main ]
+    tags: ["v*"]
```

Comment:
Hmm `v*` vs `*` in release.yml

Reply:
Also... any guarantees on relative order of this workflow vs release.yml?

Reply:
We can use [official semantic regex](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string), but it's quite long.